andela/wger_akira
# [ft 142465447] Add signals to delete muscle overview cache after a save or delete

#### What does this PR do?
Burst Exercise cache when deleting muscles
#### Description of Task to be completed?
It seems that after deleting a muscle, it keeps appearing in descriptions. This is due to the cache not being correctly reset. The cache should be reset after a muscle has been deleted and saved
#### How should this be manually tested?
Load the muscle overview page in your browser.(This ensures its cached)
Load the muscle page and add or delete a muscle.
Reload muscle overview page and see if the new muscle was added or deleted.
#### What are the relevant pivotal tracker stories?
#142465447
